conda/libmesh-vtk/build.sh

```diff
@@ -18,11 +18,13 @@ cmake .. -G "Ninja" \
     -DVTK_BUILD_TESTING:BOOL=OFF \
     -DVTK_BUILD_EXAMPLES:BOOL=OFF \
     -DBUILD_SHARED_LIBS:BOOL=ON \
-    -DVTK_USE_MPI:BOOL=ON \
     -DVTK_GROUP_ENABLE_Rendering:STRING=DONT_WANT \
     -DVTK_GROUP_ENABLE_Qt::STRING=NO \
     -DVTK_GROUP_ENABLE_Views:STRING=NO \
-    -DVTK_GROUP_ENABLE_Web:STRING=NO 
+    -DVTK_GROUP_ENABLE_Web:STRING=NO \
+    -DVTK_GROUP_ENABLE_MPI=NO \
+    -DVTK_USE_MPI:BOOL=OFF \
+    -DVTK_MODULE_ENABLE_VTK_IOMPIParallel=WANT
```

Comment:
@cticenhour this is what I was asking you for when I mentioned I was getting GLIBC issues! This _is the fix_.

The error:
```pre
nstalled/lib/libmesh_opt.so, not found (try using -rpath or -rpath-link)
Compiling C++ (in opt mode) /home/milljm/projects/moose/framework/build/unity_src/samplers_Unity.C...
/home/milljm/miniforge_testing/envs/testing/bin/../lib/gcc/x86_64-conda-linux-gnu/12.3.0/../../../../x86_64-conda-linux-gnu/bin/ld: /home/milljm/miniforge_testing/envs/testing/lib/libvtksys-9.2.so: undefined reference to `clock_gettime@GLIBC_2.17'
/home/milljm/miniforge_testing/envs/testing/bin/../lib/gcc/x86_64-conda-linux-gnu/12.3.0/../../../../x86_64-conda-linux-gnu/bin/ld: /home/milljm/miniforge_testing/envs/testing/lib/libvtkIOCore-9.2.so: undefined reference to `memcpy@GLIBC_2.14'
```

Does this make any sense that these switches would solve this issue!?